private/buf/bufformat/testdata/proto2/enum/v1/enum.golden.proto

```diff
@@ -5,12 +5,12 @@ enum Type {
   // Another trailing comment on Type.
   // A third trailing comment on Type.
 
-  TYPE_UNSPECIFIED/* Comment before '=' */ = /* Comment after '=' */0 [deprecated = true];
+  TYPE_UNSPECIFIED /* Comment before '=' */ = /* Comment after '=' */ 0 [deprecated = true];
```

Comment:
As can be seen below, the only diffs in the golden files are in whitespace around inline comments. If you look at the original files, you'll see we only expect a space in the golden output for cases where the original also had whitespace.